src/schemata/definitions/field_schema/timer.yml

```diff
@@ -1,31 +1,45 @@
 ---
 type: object
 description: |
-  Defines a field of type `timer`.
+  Defines a field of type `timer`. Timers are used to measure time or execute time-controlled actions within a Workflow Run.
+
+  Timers have two possible modes of operation: countdown and stopwatch.
+  
+  In **countdown** mode, the timer will run for a specified duration, and fire a `on_timer_complete` Trigger upon completion.
+  In **stopwatch** mode, the timer measures time until it is stopped, either manually or through a Behavior.
+  
+  Timer fields can be assigned to individual substeps by passing the field identifier to the `timer` property.
 required: [type]
 properties:
   type:
     type: string
     enum: [timer]
   defaultDuration:
-    description: A fixed duration.
+    description: |
+      The default value set on the timer upon starting the workflow. If this is defined, the timer will run in countdown mode, unless the value is reset during workflow execution.
+
+      The initial value of the timer can also be set using the `defaultValue` property. If both are defined, `defaultValue` will overwrite the value set in `defaultDuration`.
     $ref: '#/definitions/duration'
   showButtons:
     type: boolean
-    default: true
-    description: A flag indicating whether to show the buttons.
+    default: false
+    description: |
+      A flag indicating whether to show all the buttons to control the timer. When this property is set to `true`, it will take precedence over the flags for individual buttons.
```

Comment:
```suggestion
    default: true
    description: |
      A flag indicating whether to show all timer control buttons. Use this setting to conveniently remove all buttons in case of the timer being controlled entirely by [actions](/schemas/definitions/workflow_template/actions/). The settings for the individual buttons take precedence over this setting.
```

Reply:
default value is `false`. The setting takes precedence over the individual buttons